src/ocm/services/CpuArchitectureService.ts

```diff
@@ -22,8 +22,8 @@ const mapOcmArchToCpuArchitecture = (ocmArch: OcmCpuArchitecture | string): CpuA
  * Takes the cluster.cpuArchitecture value and maps it to
  * a valid infraEnv cpuArchitecture value (currently either x86 or AMD).
  *
- * As of 2022-10-27, cluster's cpuArchitecture can be:
- * - For an AI cluster: CpuArchitecture.x86, CpuArchitecture.ARM or the value "multi"
+ * As of 2023-03-06, cluster's cpuArchitecture can be:
+ * - For an AI cluster: CpuArchitecture.x86, CpuArchitecture.ARM, CpuArchitecture.ppc64le, CpuArchitecture.s390x or the value "multi"
```

Comment:
You will need to add the mapping of the new CPU architectures in the two functions `mapOcmArchToCpuArchitecture` and `mapClusterCpuArchToInfraEnvCpuArch`.